packages/prosess-tilkjent-ytelse/src/components/TilkjentYtelse.tsx

```diff
@@ -85,18 +85,28 @@ const createTooltipContent = (intl, item, getKodeverknavn, arbeidsgiverOpplysnin
 `;
 };
 
-const andelerUtgjør100ProsentTilsammen = periode => {
-  const { andeler } = periode;
-  const totalUtbetalingsgrad = (andeler || []).reduce((accumulator, andel) => accumulator + andel.utbetalingsgrad, 0);
-  if (totalUtbetalingsgrad >= 100) {
-    return true;
+const sumUtBetalingsgrad = (andeler: any) => andeler.reduce((sum, andel) => sum + andel.utbetalingsgrad, 0);
+
+const erTotalUtbetalingsgradOver100 = periode => {
+  if (periode.totalUtbetalingsgradEtterReduksjonVedTilkommetInntekt || periode.totalUtbetalingsgradFraUttak) {
```

Comment:
Denne vil vel vere false dersom både totalUtbetalingsgradEtterReduksjonVedTilkommetInntekt  og totalUtbetalingsgradFraUttak er lik 0.

Reply:
Så 0 er en gyldig verdi for disse to tallene, og denne skulle da vært true? Så det denne if'en skal sjekke at minst et av dem eksisterer?

Reply:
Ja, dersom `Math.min(periode.totalUtbetalingsgradEtterReduksjonVedTilkommetInntekt, periode.totalUtbetalingsgradFraUttak)` aksepterer at ein av parameterene er null